example/particles/particles.cpp

```diff
@@ -62,6 +62,11 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
       destroy_particles_frac >= 0. && destroy_particles_frac <= 1.,
       "Fraction of particles to destroy each timestep must be between 0 and 1");
 
+  Real defrag_threshold = pin->GetOrAddReal("Particles", "defrag_threshold", 0.9);
```

Comment:
Useful to be able to control this at runtime.

Reply:
Should we create a `parthenon/particles` input block and have this read in there? We could store this in a specialized `parthenon/particles` package or in the mesh.

(Actually I think this input block already exists... I think we now use it for boundaries... at least with #1177 )

Reply:
Defrag doesn't actually get called by default in tasklists or by the particles infrastructure itself, so I think it's reasonable to leave this up to downstream applications since they will already be deciding when/where to call Defrag when they write their tasking.